### PR TITLE
Add some files required at startup

### PR DESCRIPTION
**Pull request by minminm**
Add random dev in /dev;  add cpuinfo, meminfo and cmdline file in proc; add passwd and hosts file in /etc

### CODE REVIEW

File: modules/axfs/src/mounts.rs [hunk 1]
```diff
@@ -1,89 +1,216 @@
...
+        let mut meminfo_str = String::new();
+        meminfo_str.push_str(format!("MemTotal:       {:8}\n", totalram).as_ref());
+        meminfo_str.push_str(format!("MemFree:        {:8}\n", freeram).as_ref());
+
+        file_meminfo.write_at(0, meminfo_str.as_bytes())?;
+    }
+
+    // Create /proc/cmdline
+    proc_root.create("cmdline", VfsNodeType::File)?;
+    let file_meminfo = proc_root.clone().lookup("./cmdline")?;
```
Comment (coolyjg) [inline modules/axfs/src/mounts.rs:135]: Why `file_meminfo`? rather than `file_cmdline`?

File: modules/axfs/src/arch/x86.rs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    pub fn get_model_name(&self) -> [u8; 64] {
+        let mut model_name = [0; 64];
+        for i in 0..16 {
+            let bytes = self.x86_model_id[i].to_le_bytes();
+            model_name[i * 4..i * 4 + 4].copy_from_slice(bytes.as_ref());
+        }
+
+        log!(log::Level::Info, "{:?}", model_name);
```
Comment (coolyjg) [inline modules/axfs/src/arch/x86.rs:44]: remove this

File: modules/axfs/src/arch/x86.rs [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        /*
+         * Intel chips right-justify this string for some dumb reason;
+         * undo that brain damage:
```
Comment (coolyjg) [inline modules/axfs/src/arch/x86.rs:47]: DON'T copy others' annotations.

File: modules/axfs/src/arch/x86.rs [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+ *               http://license.coscl.org.cn/MulanPSL2
+ *   THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT, MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ *   See the Mulan PSL v2 for more details.
+ */
+
+use core::arch::x86_64::__cpuid;
+
+#[derive(Default)]
+pub struct CpuinfoX86 {
+    pub x86: u8,        /* CPU family */
```
Comment (coolyjg) [inline modules/axfs/src/arch/x86.rs:14]: why not use `x86_family`?

File: modules/axfs/src/mounts.rs [hunk 5]
```diff
@@ -1,89 +1,216 @@
...
+    file_meminfo.write_at(0, b"BOOT_IMAGE=/boot/vmlinuz-3.5.0-51-generic root=UUID=19caa20e-0443-497a-a818-7438ddc2d5d6 ro quiet splash\n")?;
```
[the remark this answers is not in the document]
Comment (minminm) [reply modules/axfs/src/mounts.rs]: not complete yet

**Comment by coolyjg** [comment]
It seems that your rebase operation is wrong. I guess you need another PR.

**Pull request closed without merging**